src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add() {
+      return 'add';
+    },
+    subtract() {
+      return 'subtract';
+    },
+    divide() {
+      return 'divide';
+    },
+    multiply() {
+      return 'multiply';
```

Comment:
change the result, instead of returning a string

Reply:
Ok, i fixed this. I hope I understood the task correctly this time. 😄